api/v1alpha1/cluster_types.go

```diff
@@ -109,6 +109,9 @@ type ETOSAPI struct {
 	ExecutionSpaceProviderSecret string `json:"executionSpaceProviderSecret"`
 	// +optional
 	LogAreaProviderSecret string `json:"logAreaProviderSecret"`
+	// +kubebuilder:default=1
+	// +optional
+	Replicas *int32 `json:"replicas"`
```

Comment:
To match the API that we are using

```suggestion
	Replicas *int32 `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
```